test/specs/apexLsp.e2e.ts

```diff
@@ -64,6 +64,10 @@ describe('Apex LSP', async () => {
     utilities.log(`${testSetup.testSuiteSuffixName} - Go to Definition`);
     // Get open text editor
     const workbench = await (await browser.getWorkbench()).wait();
+    const inputBox = await utilities.runCommandFromCommandPrompt(workbench, 'Go to File...', 1);
+    await inputBox.setText('ExampleClassTest.cls');
+    await inputBox.confirm();
+    await utilities.pause(1);
```

Comment:
This is added to every place where the E2E test switches from one tab in the editor view to another tab. It's needed because when the VSCode window is too small, some tabs are hidden to the right side and require scrolling to access. Using "Go to File" is a replacement for scrolling, which WDIO does not support.